single.php

```diff
@@ -58,36 +57,6 @@
 	get_home_url()
 );
 
-// Build the shortcode for articles block.
-if ( 'yes' === $post->include_articles ) {
-	$tag_id_array = [];
-	foreach ( $post->tags() as $post_tag ) {
-		$tag_id_array[] = $post_tag->id;
-	}
-	$category_id_array = [];
-	foreach ( $post->terms( 'category' ) as $category ) {
-		$category_id_array[] = $category->id;
-	}
-
-	$block_attributes = [
-		'exclude_post_id' => $post->ID,
-		'tags'            => $tag_id_array,
-		'post_categories' => $category_id_array,
-	];
-
-	$post->articles = '<!-- wp:planet4-blocks/articles ' . wp_json_encode( $block_attributes, JSON_UNESCAPED_SLASHES ) . ' /-->';
-}
-
-if ( ! empty( $take_action_page ) && ! has_block( 'planet4-blocks/take-action-boxout' ) ) {
```

Comment:
The Take action boxout setting was [removed](https://github.com/greenpeace/planet4-master-theme/pull/1493/files#diff-365b0310a0226ba9a9c9dee5856731efc85a084d1a373a78a6242dcdf0deb7a7L174) from post page in [PLANET-6450](https://jira.greenpeace.org/browse/PLANET-6450), removing the unused code associated with this setting.